frontend/src/scenes/pipeline/PipelineNodeNew.tsx

```diff
@@ -177,7 +179,12 @@ function nodeOptionsTable(stage: PipelineStage, targets: TableEntry[], loading:
                                 <LemonButton
                                     type="primary"
                                     data-attr={`new-${stage}-${target.id}`}
-                                    to={urls.pipelineNodeNew(stage, target.id)}
+                                    // TODO: we shouldn't load a new page, but to= errors with
+                                    // Error: Rendered fewer hooks than expected. This may be caused by an accidental early return statement
+                                    // to={urls.pipelineNodeNew(stage, target.id)}
+                                    onClick={() =>
+                                        window.open(urls.pipelineNodeNew(stage, target.id), '_self')?.focus()
```

Comment:
it would be nice to fix this, but this can be a follow-up as it's functional while annoying